scipion

```diff
@@ -32,6 +32,44 @@ import sys
 import os
 from os.path import join, exists, dirname, expanduser
 import subprocess
+import pyworkflow
```

Comment:
If 'scipion' script will be installing python environment,  probably this import will not work.
Nonetheless, probably is a good idea to separte the install logic to another script (e.g scipion-install.py or scipion-build.py) and assume here that Python environment is ready and Scipion configuration correct.

Reply:
Yep, I've been assuming for now that we have already a python (clean environment from conda or python venv) and later we can focus on what to do if we need to install python.